validator/client/src/main/java/tech/pegasys/teku/validator/client/restapi/ValidatorTypes.java

```diff
@@ -116,6 +119,48 @@
           .withField("data", listOf(REMOTE_KEY), Function.identity())
           .build();
 
+  public static DeserializableTypeDefinition<URL> URL_TYPE =
+      DeserializableTypeDefinition.string(URL.class)
+          .name("Signer")
+          .formatter(URL::toString)
+          .parser(
+              url -> {
+                try {
+                  return new URL(url);
+                } catch (MalformedURLException e) {
+                  throw new IllegalArgumentException(e);
+                }
+              })
+          .build();
```

Comment:
lets add a negative test case around this to ValidatorTypesTest to show the exception happening... should be able to just create a test that deserializes this type and uses `/` as the url...